### PR TITLE
Collection Plugin Updates

### LINKED ISSUE #3: Crashes if dashboard plugin not enabled

### LINKED ISSUE #4: Magic link Crashes if magic link plugin not enabled.

### LINKED ISSUE #5: Magic Link breaks when not logged in

### LINKED ISSUE #6: Stats on magic link show logged in users stats instead of user magic link is for

### LINKED ISSUE #7: Magic link submission Date had to be something specific for submission
No error was shown.

### LINKED ISSUE #9: Configure required plugins
https://github.com/DiscipleTools/disciple-tools-survey-collection/blob/3dd73cf1134e3e76807f19b4921bbd8e8161b9a3/admin/config-required-plugins.php#L36

### PR DESCRIPTION
**Pull request by kodinkat**
- fixes #3 
- fixes #4 
- fixes #5 
  - Unable to recreate...
- fixes #6 
- fixes #7 
- fixes #9

### CODE REVIEW

**Comment by kodinkat** [comment]
@corsacca review ready!

File: admin/config-required-plugins.php [hunk 1]
```diff
@@ -40,17 +40,31 @@
      */
     $plugins = [
         [
-            'name'                  => 'iThemes Security',
-            'slug'                  => 'better-wp-security',
-            'required'              => false,
-            'version'               => '7.2.0',
+            'name'     => 'iThemes Security',
```
Comment (corsacca) [inline admin/config-required-plugins.php:43]: @kodinkat not needed

File: admin/config-required-plugins.php [hunk 2]
```diff
@@ -40,17 +40,31 @@
...
+            'source'   => 'https://github.com/DiscipleTools/disciple-tools-dashboard/releases/latest/download/disciple-tools-dashboard.zip'
         ]
     ];
-    if ( is_multisite() ){
+    if ( is_multisite() ) {
         $plugins[] = [
-            'name' => 'Disciple.Tools Multisite Helper',
-            'slug' => 'disciple-tools-multisite',
-            'source' => 'https://github.com/DiscipleTools/disciple-tools-multisite/releases/latest/download/disciple-tools-multisite.zip',
+            'name'     => 'Disciple.Tools Multisite Helper',
```
Comment (corsacca) [inline admin/config-required-plugins.php:65]: Not needed

File: magic-link/magic-link-user-app.php [hunk 3]
```diff
@@ -1171,13 +1171,14 @@ class: "button primary",
                             Function(jsObject.submit_success_function)();
 
                         } else {
-                            jQuery('#error').html(data['message']);
+                            console.log(data);
```
Comment (corsacca) [inline magic-link/magic-link-user-app.php:1174]: log

**Review by corsacca — CHANGES_REQUESTED** [review]

**Comment by corsacca** [comment]
With plugin disabled:
image

File: magic-link/magic-link-user-app.php (hunk 3 above)
Comment (corsacca) [reply magic-link/magic-link-user-app.php:1174]: @kodinkat this one? Just no need consoling extra data.

**Review by corsacca — APPROVED** [review]

**Comment by corsacca** [comment]
thanks @kodinkat !

**Pull request merged**